src/passes/stringify-walker.h

```diff
@@ -115,15 +115,18 @@ struct HashStringifyWalker : public StringifyWalker<HashStringifyWalker> {
   // After calling walkModule, this vector contains the result of encoding a
   // wasm module as a string of uint64_t values. Each value represents either an
   // Expression or a separator to mark the end of control flow.
-  std::vector<uint64_t> hashString;
+  std::vector<unsigned> hashString;
   // A monotonic counter used to ensure that unique expressions in the
   // module are assigned a unique value in the hashString
-  uint64_t nextVal = 0;
+  unsigned nextVal = 0;
+  // A monotonic counter used to ensure that each separator in the
+  // module is assigned a unique value in the hashString
+  signed nextSeparatorVal = -1;
```

Comment:
I've never seen someone use `signed` like this 😆

Reply:
done